src/pixbuf_animation.rs

```diff
@@ -28,7 +29,10 @@ impl<T: IsA<PixbufAnimation>> PixbufAnimationExtManual for T {
         unsafe {
             from_glib_full(gdk_pixbuf_sys::gdk_pixbuf_animation_get_iter(
                 self.as_ref().to_glib_none().0,
-                start_time.as_ref().to_glib_none().0,
+                start_time
```

Comment:
Shouldn't `&start_start` be enough here?

Reply:
No, because it's an `Option`